pkg/base/testdata/raft_config

```diff
@@ -0,0 +1,20 @@
+echo
+----
+(base.RaftConfig) {
+ RaftTickInterval: (time.Duration) 200ms,
+ RaftElectionTimeoutTicks: (int) 15,
+ RaftHeartbeatIntervalTicks: (int) 5,
+ RangeLeaseRaftElectionTimeoutMultiplier: (float64) 3,
+ RangeLeaseRenewalFraction: (float64) 0.5,
+ RaftLogTruncationThreshold: (int64) 16777216,
```

Comment:
I guess we don't update these very often so it's acceptable to run into a random test failure, but mind adding a comment on RaftConfig that any changes will also need an update in this file.

Reply:
Done.